packages/kbn-dom-drag-drop/src/drag_drop.tsx

```diff
@@ -613,7 +613,11 @@ const DropsInner = memo(function DropsInner(props: DropsInnerProps) {
       onDrop: dropType ? (e: DroppableEvent) => drop(e, dropType) : noop,
       draggable,
       ghost:
-        (isActiveDropTarget && dropType !== 'reorder' && dragging?.ghost && dragging.ghost) ||
+        (true &&
```

Comment:
This was probably a temporary change which pinged Data Discovery team.

Reply:
Thanks. Didn't mean for that to sneak in!